Framework/Sources/FloatingPanel.swift

```diff
@@ -49,6 +49,11 @@ class FloatingPanel: NSObject, UIGestureRecognizerDelegate {
     var interactionInProgress: Bool = false
     var isDecelerating: Bool = false
 
+    // Animation handling
+    private var alreadyMovedTranslationOfPan: CGPoint = CGPoint.zero
+    private var totalYTranslation: CGFloat = CGFloat(0)
```

Comment:
The following should work
```suggestion
    private var totalYTranslation: CGFloat = 0
```